.eslintrc.js

```diff
@@ -2,12 +2,20 @@ module.exports = {
   root: true,
   'extends': [
     'airbnb-base',
+    'plugin:jest/all'
   ],
   plugins: [],
   env: {
     node: true,
   },
   rules: {
+    'jest/lowercase-name': ['error', { 'allowedPrefixes': ['DELETE', 'GET', 'POST', 'PUT'] }],
+    'jest/no-hooks': [
+      'error',
+      {
+        'allow': ['afterAll', 'afterEach', 'beforeAll', 'beforeEach']
+      }
+    ],
```

Comment:
Maybe `'jest/no-hooks': 'off'` since [all hooks](https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-hooks.md#allow) are accepted.

Reply:
Let me clean the tests base not to have this rule ;)